source/render/ODFrameListener.cpp

```diff
@@ -278,9 +278,26 @@ void ODFrameListener::setCameraPosition(const Ogre::Vector3& position)
     mCameraManager.setCameraPosition(position);
 }
 
-void ODFrameListener::moveCamera(CameraManager::Direction direction)
+void ODFrameListener::moveCamera(CameraManager::Direction direction, double aux)
 {
-    mCameraManager.move(direction);
+    mCameraManager.move(direction, aux);
+}
+// Code below stopped working after removing methods from CameraManager; I'm leaving  
```

Comment:
aux - auxilery value for move method. For randomRotateX it is the distance of the mouse moved,
0 as default.